.github/workflows/extension_upgrade.yml

```diff
@@ -66,6 +66,7 @@ jobs:
           cargo install cargo-pgrx --version 0.11.0
           SQLX_OFFLINE=true cargo pgrx run pg16 --pgcli || true
           export DATABASE_URL=postgres://$USER:postgres@localhost:28816/pgmq
+          psql $DATABASE_URL -c "DROP SCHEMA IF EXISTS pgmq CASCADE;"
```

Comment:
@v0idpwn , im sneaking a patch in for the extension upgrade tests. It is really hard to reproduce, but I am pretty sure the issue with failing upgrade was that the "SQL only" tests were leaving objects lingering around. `DROP EXTENSION` wouldnt clear out those objects, but `DROP SCHEMA` does.

Reply:
I merged this in https://github.com/pgmq/pgmq/pull/436 instead.